packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -102,57 +114,69 @@ export const IndexesTable: React.FunctionComponent<IndexesTableProps> = ({
     return _columns;
   }, [canDeleteIndex, onSortTable]);
 
+  useEffect(() => {
+    const container =
+      containerRef.current?.getElementsByTagName('table')[0]?.parentElement;
+    if (container) {
+      container.style.height = `${scrollHeight}px`;
+    }
```

Comment:
Please let's add a detailed comment here explaining what and why we are doing, as this is kinda obscure workaround for an issue with leafygreen components I worry that this runs the risk of being removed or broken at some point in the future due to lost context